internal/clients/telegram/state/response_strings.go

```diff
@@ -0,0 +1,33 @@
+package state
+
+import "github.com/m-kuzmin/daily-reporter/internal/template"
+
+/*
+Responses holds parsed and ready to use responses for all states. You can be sure no state uses a response not in this
+struct because of `ConversationState` interface.
+*/
+type Responses struct {
+	Root      rootResponses
+	AddAPIKey addAPIKeyResponses
+}
+
+/*
+NewResponses populates response objects from template. This ensures all strings and groups are in the template
+in one place (this func).
+*/
+func NewResponses(templ template.Template) (Responses, error) {
+	root, err := newRootResponses(templ)
+	if err != nil {
+		return Responses{}, err
+	}
+
+	addAPIKey, err := newAddAPIKeyResponse(templ)
```

Comment:
Since the constructors are very similar could use reflection as in `template.Group.Populate()`.